src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt

```diff
@@ -72,40 +99,40 @@ class MarkdownProjectReader() : IProjectReader {
             )
         }
     }
+
```

Comment:
Please undo the function reordering; it makes it harder to see which lines have changed.